### PR TITLE
Expand GitHub Actions tests

### PR DESCRIPTION
**Pull request by j-adamczyk**

### CODE REVIEW

**Review by my-alaska — APPROVED** [review]

File: .github/workflows/python-test.yml [hunk 1]
```diff
@@ -1,40 +1,49 @@
...
+      - uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
 
-    - name: Set up Python
-      uses: actions/setup-python@v4
-      with:
-        python-version: '3.9'
+      - name: Install Poetry
+        uses: abatilo/actions-poetry@v2
```
Comment (mjste) [inline .github/workflows/python-test.yml:26]: Version 3 is available. Can we use it?

Comment (mjste) [inline .github/workflows/python-test.yml:26]: We can probably add sort plugin here as a parameter to this action instead of our own step

Comment (j-adamczyk) [reply .github/workflows/python-test.yml:26]: Sure

Comment (j-adamczyk) [reply .github/workflows/python-test.yml:26]: Good idea

**Review by mjste — APPROVED** [review]

**Review by mjste — APPROVED** [review]

**Pull request merged**